core/trino-main/src/main/java/io/trino/sql/planner/planprinter/PlanPrinter.java

```diff
@@ -1694,46 +1775,23 @@ public NodeRepresentation addNode(
         }
     }
 
-    private static String formatFrame(WindowNode.Frame frame)
+    private static <T> String formatCollection(Collection<T> collection)
     {
-        StringBuilder builder = new StringBuilder(frame.getType().toString());
-
-        frame.getOriginalStartValue().ifPresent(value -> builder.append(" ").append(value));
-        builder.append(" ").append(frame.getStartType());
-
-        frame.getOriginalEndValue().ifPresent(value -> builder.append(" ").append(value));
-        builder.append(" ").append(frame.getEndType());
-
-        return builder.toString();
-    }
-
-    @SafeVarargs
-    private static String formatHash(Optional<Symbol>... hashes)
-    {
-        List<Symbol> symbols = stream(hashes)
-                .filter(Optional::isPresent)
-                .map(Optional::get)
-                .collect(toList());
-
-        if (symbols.isEmpty()) {
+        if (collection.isEmpty()) {
             return "";
         }
-
-        return "[" + Joiner.on(", ").join(symbols) + "]";
-    }
-
-    private static String formatOutputs(TypeProvider types, Iterable<Symbol> outputs)
-    {
-        return Streams.stream(outputs)
-                .map(input -> input + ":" + types.get(input).getDisplayName())
-                .collect(Collectors.joining(", ", "[", "]"));
+        return collection.stream()
+                .map(Objects::toString)
+                .collect(joining(", ", "[", "]"));
     }
 
-    public static String formatAggregation(Aggregation aggregation)
+    public static String formatAggregation(Anonymizer anonymizer, Aggregation aggregation)
```

Comment:
nit: consider making it non static so that you don't have to pass `anonymizer`

Reply:
It'll be redundant to make `formatAggregation` static because it is being used in `GraphVizPlanPrinter` too.

Reply:
Anyways in `PlanPrinter`, `formatAggregation` is only used at 2-3 places.